jinx.el

```diff
@@ -858,6 +922,17 @@ If prefix argument ALL non-nil correct all misspellings."
         (goto-char old-point)
         (jinx--in-base-buffer #'jit-lock-refontify (window-start) (window-end))))))
 
+;;;###autoload
+(defun jinx-correct-at-point ()
+  "Correct word at cursor.
+Suggest corrections even if it's not misspelled."
+  (interactive)
+  (unless jinx-mode (jinx-mode 1))
+  (pcase-let ((`(,beg . ,end) (jinx--bounds-of-word-at-point)))
+    (if (and beg end)
+        (jinx--correct beg end)
+      (user-error "No word at point"))))
```

Comment:
What do you think about the approach proposed in https://github.com/minad/jinx/pull/89#issuecomment-1620125749? If we go that way, you wouldn't have to change all the arguments from OV to BEG+END.